ODIN_II/SRC/simulate_blif.cpp

```diff
@@ -2222,10 +2222,12 @@ static int verify_test_vector_headers(FILE* in, lines_t* l) {
         error_message(SIMULATION, 0, -1, "%s\n", "Failed to read vector headers.");
```

Comment:
read_buffer gets it value from this function here and we see it's defined as a char array.